End-to-End-Tests.md

```diff
@@ -176,6 +180,66 @@ $ python -m scripts.run_e2e_tests --suite="suiteName"
 
 Chrome will open and start running your tests.
 
+### Set chromedriver version
+
+The end-to-end tests are run on the Chrome browser and depend on chromedriver. The chromedriver version to be used depends on the Chrome browser version installed on the machine. We try to determine this version automatically, but if our automatic determination fails, you'll see an error with this advice:
+
+```text
+Please set the chromedriver version manually using --chrome_driver_version flag.
```

Comment:
Should this error message have "using the" instead of "using"? Maybe we can fix the original if it's wrong.

Reply:
The code in oppia/oppia is wrong. I'm opening a PR to fix it

Reply:
done